applicationRoutes.js

```diff
@@ -43,7 +43,17 @@ router.get('/faq', (req, res) => {
   });
 });
 
+const linkPartners = [
```

Comment:
Can we pull out the various bits where we'd want a partner to change stuff (so the referer allow-list and the icon) into a separate file so it's very easy to know it's 3rd party config？

Reply:
I think that's a good idea.